apps/.eslintrc.js

```diff
@@ -164,13 +165,29 @@ module.exports = {
             message: 'Use jest matchers instead of chai',
           },
           {
-            name: 'sinon',
-            message: 'Use jest spys and stubs instead of sinon',
+            name: 'enzyme',
```

Comment:
Should we remove this from the global [eslintrc](https://github.com/code-dot-org/code-dot-org/blob/staging/.eslintrc.js#L36) then?

Reply:
nice catch! yes, removed.

it appears the `no-restricted-imports` rule there has no effect within `apps` because the same rule is specified in `apps/.eslintrc.js`. I do see instances of `sinon` in `aws`, but we don't appear to be running lint there because they are not accompanied by `eslint-disable` directives: https://github.com/code-dot-org/code-dot-org/blob/331569dbddbd5cf4aac46f36c825d5443093f384/aws/cloudformation/test/testHoneybadgerNotify.js#L1-L3